spopt/tests/test_locate.py

```diff
@@ -11,6 +11,12 @@
 import unittest
 import os
 import pickle
+import platform
+
+if os == "windows":
```

Comment:
You'll need to [define `os` as the string of the operating system](https://github.com/jGaboardi/tigernet/blob/be2724e662b183a3016b4bf6a0b18c34a8f51985/tigernet/tests/test_tigernet_empirical_gdf.py#L13-L16). Currently, it is set as the `os` package. Are we using the `os` python package anywhere else here? If so, we can simply rename the variable like so:

```python
import platform

operating_system = platform.platform()[:7].lower()
if operating_system == "windows":
    WINDOWS = True
else:
    WINDOWS = False
```